### PR TITLE
chore(planning/control packages): organized authors and maintainers

### PR DESCRIPTION
**Pull request by takayuki5168**
## Description

organized planning/control authors and maintainers
## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by takayuki5168** [comment]
@TakaHoribe I've finished the fix for colcon test.

File: control/external_cmd_selector/package.xml [hunk 1]
```diff
@@ -6,6 +6,7 @@
   <description>The external_cmd_selector package</description>
   <maintainer email="kenji.miyake@tier4.jp">Kenji Miyake</maintainer>
   <license>Apache License 2.0</license>
+  <author email="kenji.miyake@tier4.jp">Kenji Miyake</author>
 
   <author email="kenji.miyake@tier4.jp">Kenji Miyake</author>
```
Comment (kenji-miyake) [inline control/external_cmd_selector/package.xml:11]: Seems duplicated.

File: control/external_cmd_selector/package.xml [hunk 2]
```diff
@@ -6,6 +6,7 @@
   <description>The external_cmd_selector package</description>
   <maintainer email="kenji.miyake@tier4.jp">Kenji Miyake</maintainer>
   <license>Apache License 2.0</license>
+  <author email="kenji.miyake@tier4.jp">Kenji Miyake</author>
```
Comment (takayuki5168) [inline control/external_cmd_selector/package.xml:9]:
```suggestion
```

File: control/external_cmd_selector/package.xml (hunk 1 above)
Comment (takayuki5168) [reply control/external_cmd_selector/package.xml]: https://github.com/autowarefoundation/autoware.universe/pull/1610/commits/a9c34db8ca634f68de81326721ffad315c0ba537
DONE

File: control/vehicle_cmd_gate/package.xml [hunk 3]
```diff
@@ -4,9 +4,11 @@
   <name>vehicle_cmd_gate</name>
   <version>0.1.0</version>
   <description>The vehicle_cmd_gate package</description>
-  <maintainer email="h_ohta@ertl.jp">h_ohta</maintainer>
+  <maintainer email="takamasa.horibe@tier4.jp">Takamasa Horibe</maintainer>
   <license>Apache License 2.0</license>
 
+  <author email="h_ohta@ertl.jp">h_ohta</author>
```
Comment (kenji-miyake) [inline control/vehicle_cmd_gate/package.xml:10]: To be the same as the following line. cc @h-ohta 
https://github.com/autowarefoundation/autoware.universe/blob/a8a58c045e718d6e3193e9521adf2ba9eaad7ce2/common/tier4_state_rviz_plugin/package.xml#L7


```suggestion
  <author email="hiroki.ota@tier4.jp">Hiroki OTA</author>
```

File: planning/freespace_planner/package.xml [hunk 4]
```diff
@@ -4,9 +4,10 @@
   <name>freespace_planner</name>
   <version>0.1.0</version>
   <description>The freespace_planner package</description>
-  <maintainer email="kenji.miyake@tier4.jp">Kenji Miyake</maintainer>
-  <maintainer email="aohsato@gmail.com">Akihito OHSATO</maintainer>
+  <maintainer email="takamasa.horibe@tier4.jp">Takamasa Horibe</maintainer>
   <license>Apache License 2.0</license>
+  <author email="kenji.miyake@tier4.jp">Kenji Miyake</author>
+  <author email="aohsato@gmail.com">Akihito OHSATO</author>
```
Comment (kenji-miyake) [inline planning/freespace_planner/package.xml:10]: @aohsato Please confirm this line.

File: planning/freespace_planning_algorithms/package.xml [hunk 5]
```diff
@@ -4,8 +4,9 @@
   <name>freespace_planning_algorithms</name>
   <version>0.1.0</version>
   <description>The freespace_planning_algorithms package</description>
-  <maintainer email="aohsato@gmail.com">Akihito Ohsato</maintainer>
+  <maintainer email="takamasa.horibe@tier4.jp">Takamasa Horibe</maintainer>
   <license>Apache License 2.0</license>
+  <author email="aohsato@gmail.com">Akihito Ohsato</author>
```
Comment (kenji-miyake) [inline planning/freespace_planning_algorithms/package.xml:9]: @aohsato Same here.

File: planning/motion_velocity_smoother/package.xml [hunk 6]
```diff
@@ -4,11 +4,15 @@
   <name>motion_velocity_smoother</name>
   <version>0.1.0</version>
   <description>The motion_velocity_smoother package</description>
-  <maintainer email="horibe.takamasa@gmail.com">Takamasa Horibe</maintainer>
+
   <maintainer email="fumiya.watanabe@tier4.jp">Fumiya Watanabe</maintainer>
-  <maintainer email="yutaka.shimizu@tier4.jp">Yutaka Shimizu</maintainer>
   <license>Apache License 2.0</license>
 
+  <author email="horibe.takamasa@gmail.com">Takamasa Horibe</author>
```
Comment (kenji-miyake) [inline planning/motion_velocity_smoother/package.xml:11]:
```suggestion
  <author email="takamasa.horibe@tier4.jp">Takamasa Horibe</author>
```

File: planning/planning_debug_tools/package.xml [hunk 7]
```diff
@@ -6,6 +6,7 @@
   <description>The planning_debug_tools package</description>
   <maintainer email="horibe.takamasa@gmail.com">Takamasa Horibe</maintainer>
   <license>Apache License 2.0</license>
+  <author email="horibe.takamasa@gmail.com">Takamasa Horibe</author>
```
Comment (kenji-miyake) [inline planning/planning_debug_tools/package.xml:9]:
```suggestion
  <author email="takamasa.horibe@tier4.jp">Takamasa Horibe</author>
```

File: control/operation_mode_transition_manager/package.xml [hunk 8]
```diff
@@ -4,6 +4,7 @@
   <description>The vehicle_cmd_gate package</description>
   <maintainer email="horibe.takamasa@gmail.com">Takamasa Horibe</maintainer>
```
Comment (kenji-miyake) [inline control/operation_mode_transition_manager/package.xml:5]:
```suggestion
  <maintainer email="takamasa.horibe@tier4.jp">Takamasa Horibe</maintainer>
```

File: control/pure_pursuit/package.xml [hunk 9]
```diff
@@ -4,10 +4,12 @@
   <name>pure_pursuit</name>
   <version>0.1.0</version>
   <description>The pure_pursuit package</description>
-  <maintainer email="kenji.miyake@tier4.jp">Kenji Miyake</maintainer>
   <maintainer email="horibe.takamasa@gmail.com">Takamasa Horibe</maintainer>
```
Comment (kenji-miyake) [inline control/pure_pursuit/package.xml:7]:
```suggestion
  <maintainer email="takamasa.horibe@tier4.jp">Takamasa Horibe</maintainer>
```

File: control/shift_decider/package.xml [hunk 10]
```diff
@@ -6,6 +6,7 @@
   <description>The shift_decider package</description>
   <maintainer email="horibe.takamasa@gmail.com">Takamasa Horibe</maintainer>
```
Comment (kenji-miyake) [inline control/shift_decider/package.xml:7]:
```suggestion
  <maintainer email="takamasa.horibe@tier4.jp">Takamasa Horibe</maintainer>
```

File: planning/freespace_planner/package.xml [hunk 11]
```diff
@@ -4,9 +4,10 @@
...
 
   <author email="aohsato@gmail.com">Akihito OHSATO</author>
```
Comment (kenji-miyake) [inline planning/freespace_planner/package.xml:12]: @aohsato Same here. Could you submit a PR to replace your email address? :pray:

File: planning/planning_debug_tools/package.xml [hunk 12]
```diff
@@ -6,6 +6,7 @@
   <description>The planning_debug_tools package</description>
   <maintainer email="horibe.takamasa@gmail.com">Takamasa Horibe</maintainer>
```
Comment (kenji-miyake) [inline planning/planning_debug_tools/package.xml:7]:
```suggestion
  <maintainer email="takamasa.horibe@tier4.jp">Takamasa Horibe</maintainer>
```

File: control/shift_decider/package.xml [hunk 13]
```diff
@@ -6,6 +6,7 @@
...
   <license>Apache License 2.0</license>
+  <author email="horibe.takamasa@gmail.com">Takamasa Horibe</author>
 
   <author email="horibe.takamasa@gmail.com">Takamasa Horibe</author>
```
Comment (kenji-miyake) [inline control/shift_decider/package.xml:11]: Duplicated.

Comment (kenji-miyake) [inline control/shift_decider/package.xml:11]:
```suggestion
  <author email="takamasa.horibe@tier4.jp">Takamasa Horibe</author>
```

File: control/pure_pursuit/package.xml [hunk 14]
```diff
@@ -4,10 +4,12 @@
...
   <license>Apache License 2.0</license>
 
+  <author email="kenji.miyake@tier4.jp">Kenji Miyake</author>
+  <author email="horibe.takamasa@gmail.com">Takamasa Horibe</author>
```
Comment (kenji-miyake) [inline control/pure_pursuit/package.xml:11]:
```suggestion

  <author email="kenji.miyake@tier4.jp">Kenji Miyake</author>
  <author email="takamasa.horibe@tier4.jp">Takamasa Horibe</author>
```

File: control/operation_mode_transition_manager/package.xml [hunk 15]
```diff
@@ -4,6 +4,7 @@
...
   <license>Apache License 2.0</license>
+  <author email="horibe.takamasa@gmail.com">Takamasa Horibe</author>
```
Comment (kenji-miyake) [inline control/operation_mode_transition_manager/package.xml:7]:
```suggestion
  <author email="takamasa.horibe@tier4.jp">Takamasa Horibe</author>
```

File: planning/freespace_planner/package.xml [hunk 16]
```diff
@@ -4,9 +4,10 @@
...
   <author email="antm678@ertl.jp">Tomohito ANDO</author>
```
Comment (h-ohta) [inline planning/freespace_planner/package.xml:13]:
```suggestion
  <author email="tomohito.ando@tier4.jp">Tomohito ANDO</author>
```

Comment (h-ohta) [reply planning/freespace_planner/package.xml:13]: cc: @TomohitoAndo

**Review by kenji-miyake — APPROVED** [review]

**Pull request merged**